service/committer/committer/committer.go

```diff
@@ -138,6 +144,10 @@ func (c *Committer) Run() {
 	}
 }
 
+func (c *Committer) Shutdown() {
```

Comment:
I think we should explicitly release the connection by calling `db.Close`, `redis.Close` ... in the close function

Reply:
yes, you are right. let me update it.